test/units/modules/system/test_java_keystore.py

```diff
@@ -95,12 +95,13 @@ def test_create_jks_fail_export_pkcs12(self):
 
         with patch('os.remove', return_value=True):
             self.run_commands.side_effect = [(1, '', ''), (0, '', '')]
-            create_jks(module, "test", "openssl", "keytool", "/path/to/keystore.jks", "changeit")
+            create_jks(module, "test", "openssl", "keytool", "/path/to/keystore.jks", "changeit", "")
```

Comment:
Could you please modify or add a test where there's a passphrase (to make sure that the command contains `-passin 'pass:mypassphrase'`)

Reply:
I will work on it.